src/main/java/org/spongepowered/common/SpongeImplHooks.java

```diff
@@ -288,7 +288,7 @@ public static void onTileEntityError(TileEntity tileEntity, CrashReport crashRep
     }
 
     public static void blockExploded(Block block, World world, BlockPos blockpos, Explosion explosion) {
-        world.setBlockState(blockpos, Blocks.AIR.getDefaultState(), 2);
```

Comment:
Vanilla source uses 
`world.setBlockState(blockpos, Blocks.AIR.getDefaultState(), 3);`
flag 3 being notify client and server, any reason you are using `setBlockToAir` instead?

Reply:
`setBlockToAir` does the same thing, just a preference in fact.